charts/library/common/templates/classes/_secret.tpl

```diff
@@ -0,0 +1,34 @@
+{{/*
+This template serves as a blueprint for all Secret objects that are created
+within the common library.
+*/}}
+{{- define "bjw-s.common.class.secret" -}}
```

Comment:
same with this one, will charts switching from k8s-at-home to this one need to use this new ... namespace?